app/policies/aal3_policy.rb

```diff
@@ -32,13 +37,15 @@ def aal3_configured_and_required?
   private
 
   def aal3_requested?
+    return false if @hspd12_requested
```

Comment:
this seems counterintuitive to me.... this is because HSPD12 is a separate object?

I feel like since HSPD12 is a subset of AAL3....that if we just made `AUTH_METHODS` in `#aal3_used?` dynamic, we could combine these, right?

Reply:
I think we could bring over `piv_cac_required?` into this object as well?